frontend/app/src/components/price-manager/historic/HistoricPriceTable.vue

```diff
@@ -45,34 +54,30 @@
 <script setup lang="ts">
 import { NotificationPayload, Severity } from '@rotki/common/lib/messages';
 import { get, set } from '@vueuse/core';
-import { computed, onMounted, PropType, ref, toRef, watch } from 'vue';
+import { computed, onMounted, PropType, ref, watch } from 'vue';
 import { useI18n } from 'vue-i18n-composable';
```

Comment:
nitpick: Except the `PropType` we can now remove the other imports since auto-imports works fine